flagsmith-engine/features/models.ts

```diff
@@ -0,0 +1,105 @@
+import { v4 as uuidv4 } from 'uuid';
+import { getHashedPercentateForObjIds } from '../utils/hashing';
+
+export class FeatureModel {
+    id: number;
+    name: string;
+    type: string;
+
+    constructor(id: number, name: string, type: string) {
+        this.id = id;
+        this.name = name;
+        this.type = type;
+    }
+
+    eq(other: FeatureModel) {
+        return !!other && this.id === other.id;
+    }
+}
+
+export class MultivariateFeatureOptionModel {
+    value: any;
+    id: number | undefined;
+
+    constructor(value: any, id?: number) {
+        this.value = value;
+        this.id = id;
+    }
+}
+
+export class MultivariateFeatureStateValueModel {
+    multivariateFeatureOption: MultivariateFeatureOptionModel;
+    percentageAllocation: number;
+    id: number;
+    mvFsValueUuid: string = uuidv4();
+
+    constructor(
+        multivariate_feature_option: MultivariateFeatureOptionModel,
+        percentage_allocation: number,
+        id: number,
+        mvFsValueUuid?: string
+    ) {
+        this.id = id;
+        this.percentageAllocation = percentage_allocation;
+        this.multivariateFeatureOption = multivariate_feature_option;
+        this.mvFsValueUuid = mvFsValueUuid || this.mvFsValueUuid;
+    }
+}
+
+export class FeatureStateModel {
+    feature: FeatureModel;
+    enabled: boolean;
+    djangoID: number;
+    featurestateUUID: string = uuidv4();
+    private value: any;
+    multivariateFeatureStateValues: MultivariateFeatureStateValueModel[] = [];
+
+    constructor(
+        feature: FeatureModel,
+        enabled: boolean,
+        djangoID: number,
+        value?: any,
+        featurestateUuid: string = uuidv4()
+    ) {
+        this.feature = feature;
+        this.enabled = enabled;
+        this.djangoID = djangoID;
+        this.value = value;
+        this.featurestateUUID = featurestateUuid;
+    }
+
+    setValue(value: any) {
+        this.value = value;
+    }
+
+    getValue(identityId?: number | string) {
+        if (!!identityId && this.multivariateFeatureStateValues.length > 0) {
+            return this.getMultivariateValue(identityId);
+        }
+        return this.value;
+    }
+
+    get_feature_state_value() {
```

Comment:
As far as I can tell, this method can be removed. I think it was a hangover from before I split out the Django schemas in the python engine, I've removed it from the python engine [here](https://github.com/Flagsmith/flagsmith-engine/pull/102) and all seems well. You can replace any usages in the tests with direct calls to the `getValue` method above.